x-pack/plugins/synthetics/server/saved_objects/saved_objects.ts

```diff
@@ -46,7 +46,71 @@ export const registerUptimeSavedObjects = (
   encryptedSavedObjects.registerType({
     type: syntheticsServiceApiKey.name,
     attributesToEncrypt: new Set(['apiKey']),
-    attributesToExcludeFromAAD: new Set([ConfigKey.ALERT_CONFIG]),
+    attributesToIncludeInAAD: new Set([
```

Comment:
@elastic/obs-ux-infra_services-team I think this list might need some attention. I noticed that some of the items are nested (e.g. 'check.response.body.negative').  If this list does not represent a complete picture of the object, we will have to make some substitutions (e.g. 'check.response.body.negative' -> 'check') to properly construct AAD to match the previous implementation.